README.md

```diff
@@ -26,6 +26,7 @@ When you get started with The Things Network, you'll probably have some question
   On a fresh installation you might need to install the [MQTT plugin](https://www.rabbitmq.com/mqtt.html).  
   If you're on Linux, you probably know how to do that. On a Mac, just run `brew bundle`. The Windows installer will setup and start RabbitMQ as a service. Use the `RabbitMQ Command Prompt (sbin dir)` to run commands, i.e. to enable plugins.
 5. Declare a RabbitMQ exchange `ttn.handler` of type `topic`. Using [the management plugin](http://www.rabbitmq.com/management.html), declare the exchange in the web interface `http://server-name:15672` or using the management cli, run `rabbitmqadmin declare exchange name=ttn.handler type=topic auto_delete=false durable=true`
+6. You need make. For windows you can get it from [https://gnuarmeclipse.github.io/windows-build-tools/](https://gnuarmeclipse.github.io/windows-build-tools/)
```

Comment:
- I think this should go higher on the list. 
- Explain that on Linux you have to install `build-essential`. On macOS, `make` comes with XCode or the developer tools.
- Use backticks for formatting of `make`